src/pages/library/Datasets.js

```diff
@@ -115,9 +116,23 @@ const NIHCommonsButtons = () => buttonPrimary({
   tooltip: browseTooltip
 }, ['Browse STAGE Repository'])
 
-const thousandGenomes = () => h(Participant, {
+const thousandGenomesHighCoverage = () => h(Participant, {
+  logo: { src: thousandGenomesAnvil, alt: '1000 Genomes and AnVIL' },
```

Comment:
I could imagine changing the `logo` property to be a component instead of an object. Uses other than this one could make use of a function, something like `standardLogo` to create the image component; that would keep the styling in one place as it currently is. Then this case could construct a flexbox to pass in instead of the single image.

Reply:
I got updated logos from Lou, thoughts?